config.json

```diff
@@ -534,6 +534,14 @@
         "practices": [],
         "prerequisites": [],
         "difficulty": 8
+      },
+      {
+        "slug": "automated-readability-index",
+        "name": "Automated readability index",
+        "uuid": "ccf7118a-f2b7-47ca-b7a1-18e830daca2f",
+        "practices": [],
+        "prerequisites": [],
+        "difficulty": 8
```

Comment:
I think 8 is too high. How about 4?

Reply:
we can reduce. there is an idea to add another task as a continuation of this where it will be possible to select a specific calculation formula. some calculation methods require counting syllables And this is quite difficult